src/components/events/EventsList/EventCard.tsx

```diff
@@ -28,26 +30,31 @@ const EventCard = ({ event }: { event: EventItem }) => {
           blurDataURL={
             "https://res.cloudinary.com/tiyeni/image/upload/c_scale,h_678,q_auto:low/v1679808591/2X0A4983.jpg"
           }
-          src={event.coverImg}
+          src={event.imageUrl ?? defaultImage.asset.url}
           alt={event.title}
           className="absolute !-top-[10%] !h-[110%] w-full object-cover"
         />
         <div className="relative flex items-end p-8">
           <div className="flex items-center text-white">
-            <span className="mr-2 text-lg font-bold">{event.type}</span>{" "}
+            <span className="mr-2 text-lg font-bold">
+              {event.audience.name}
+            </span>{" "}
             <BookOpenIcon className="h-6" />
           </div>
         </div>
       </div>
       <div className="flex items-center justify-end">
-        <div className="mr-4 text-xl font-semibold text-secondary">
-          23 Places Left
+        {
+          event.spotCounter && <div className="mr-4 text-xl font-semibold text-secondary">
+          {event.spotCounter} Places Left
         </div>
-        <div>
+        }
+        
+        <Link href={`/events/${event._id}`}>
```

Comment:
the register button should show up if the event has a registration link, otherwise it should be `view more`